src/models/device.ts

```diff
@@ -149,12 +149,17 @@ const getDeviceModel = function (
 		}),
 	);
 
-	const getOsUpdateHelper = once(async () => {
+	const getOsUpdateHelper = async (deviceActionsApiVersion: 'v1' | 'v2') => {
```

Comment:
If we remove the call to `once`, it doesn't make sense to retain getOsUpdateHelper because it doesn't really provide any value. The problem is that we need to dynamically pass the API version when actually running the startOsUpdate() or getOsUpdateStatus() functions. Well, we already pass in uuid dynamically to those functions, so let's pass apiVersion the same way.

So to retain the `once`, remove the API version from util.device-actions.os-update.getOsUpdateHelper(). Instead add apiVersion as a parameter to the startOsUpdate() function in that module, just like uuid. Modify the callers of that function to pass in the API version at that time. 

You don't need to add an apiVersion to the getOsUpdateStatus() function in util.device-actions.os-update because there is not a v2, and the SDK user does not specify `runDetached`. Actually, I think this change means the src.models.getOsUpdateStatus() doesn't change at all in this PR!

This way we keep the runtime performance the same at essentially no extra cost. Every time those functions are called, they end up calling DeviceActionService startAction() and getActionStatus() which dynamically compute the URL for the action server endpoint anyway.

Please create a new commit for this change so we can easily review the diff. We can squash later.